editor/editor_inspector.cpp

```diff
@@ -3114,7 +3114,18 @@ void EditorInspector::update_tree() {
 			// Build the doc hint, to use as tooltip.
 
 			// Get the class name.
-			StringName classname = doc_name == "" ? object->get_class_name() : doc_name;
+			StringName classname = doc_name;
+			if (classname == "") {
+				classname = object->get_class_name();
+				Resource *res = Object::cast_to<Resource>(object);
+				if (res && object->get_script()) {
+					Script *src = Object::cast_to<Script>(res->get_script());
+					if (src) {
```

Comment:
```suggestion
				if (res && res->get_script().is_valid()) {
```

Reply:
`res->get_script()`evaluates to a Variant, as such there is no `is_valid()`.
I looked at the sources and saw that variant has `Variant::is_null()` and it seems to me to apply to this particular situation.